public/big-icon.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="536" height="523" viewBox="0 0 536 523" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M116 61H428M271.987 174.602C308.074 174.601 343.046 187.129 370.943 210.053C398.84 232.976 417.936 264.877 424.979 300.319C432.021 335.761 426.574 372.551 409.565 404.422C392.556 436.293 365.037 461.273 331.698 475.105C298.358 488.937 261.261 490.765 226.726 480.279C192.192 469.792 162.356 447.64 142.304 417.596C122.251 387.551 113.222 351.474 116.754 315.511C120.287 279.548 136.162 245.924 161.676 220.367C176.146 205.838 193.34 194.316 212.271 186.462C231.201 178.608 251.495 174.577 271.987 174.602V174.602Z" stroke="url(#paint0_linear)" stroke-width="71" stroke-linecap="round" stroke-linejoin="round"/>
+<rect width="536" height="518" fill="url(#paint1_linear)"/>
+<defs>
+<linearGradient id="paint0_linear" x1="91.3364" y1="409.144" x2="453.129" y2="47.8714" gradientUnits="userSpaceOnUse">
+<stop stop-color="#FFC58F"/>
+<stop offset="1" stop-color="#6FE5FF"/>
+</linearGradient>
+<linearGradient id="paint1_linear" x1="446" y1="80" x2="20" y2="432" gradientUnits="userSpaceOnUse">
+<stop stop-color="white"/>
+<stop offset="1" stop-color="white" stop-opacity="0"/>
+</linearGradient>
+</defs>
+</svg>
```

Comment:
It looks like we are using `big-icon.png` over this SVG format, is that correct? If we are using the `.png` file, can we delete this one?